backend/clubs/views.py

```diff
@@ -1201,6 +1202,31 @@ def create(self, request, *args, **kwargs):
                 f"please email {settings.FROM_EMAIL} if this is en error."
             )
 
+        if request.data.get("is_recurring", None) is not None:
+            parent_recurring_event = RecurringEvent.objects.create()
+            event_data = request.data.copy()
+            start_time = parse(event_data.pop("start_time"))
+            end_time = parse(event_data.pop("end_time"))
+            offset = event_data.pop("offset")
+            end_date = parse(event_data.pop("end_date"))
+            event_data.pop("is_recurring")
+            club_code = kwargs["club_code"]
+
+            club = Club.objects.get(code=club_code)
+            while start_time < end_date:
+                Event.objects.create(
+                    **event_data,
+                    creator=request.user,
+                    club=club,
+                    start_time=start_time,
+                    end_time=end_time,
+                    parent_recurring_event=parent_recurring_event,
+                )
+                start_time = start_time + datetime.timedelta(days=offset)
+                end_time = end_time + datetime.timedelta(days=offset)
+
+            return Response({"success": True})
```

Comment:
A potentially better response for this would be to return the serialized list of created clubs. This can be added in a future PR.

Reply:
This is also ~3-5 lines of code, so it wouldn't be too hard to add to this PR.